frontend/src/pages/SchedulePage.tsx

```diff
@@ -0,0 +1,104 @@
+import React, {ChangeEvent, FormEvent, useContext, useEffect, useMemo, useState} from "react";
+import SeriesController from "../controllers/SeriesController";
+import {Series} from "../models/Series";
+import {PaymentDTO} from "../models/Payment";
+import SeriesItems from "../components/SeriesItems";
+import {ICategoryController, ISeriesController} from "../models/ControllerTypes";
+import {AuthContext} from "../context/AuthProvider";
+import './SchedulingPage.scss'
+import CategoryController from "../controllers/CategoryController";
+import {Category} from "../models/Category";
+import {DateRangePicker, LocalizationProvider} from "@mui/lab";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import {Box, TextField} from "@mui/material";
+import {DateRange} from "@mui/lab/DateRangePicker/RangeTypes";
+
+interface SelectInput {
+    selectCategory: {value: string}
+}
+
+export default function SchedulePage() {
+    const config = useContext(AuthContext).config
+    const [series, setSeries] = useState<Series[]>([])
+    const [scheduledDate, setScheduledDate] = useState<number>(1)
+    const [description, setDescription] = useState<string>("")
+    const [amount, setAmount] = useState<number>(1)
+    const [categories, setCategories] = useState<Category[]>([])
+    const [rangeValue, setRangeValue] = useState<DateRange<Date>>([null, null])
+    const seriesController: ISeriesController = useMemo(() => SeriesController(config), [config])
+    const categoryController: ICategoryController = useMemo(() => CategoryController(config), [config])
+
+    useEffect(() => {
+        seriesController.getSeries().then(setSeries)
+        categoryController.getCategories().then(setCategories)
+    }, [seriesController, categoryController])
+
+    const addSeries = (event: FormEvent<HTMLFormElement>) => {
+        event.preventDefault()
+        const form = event.currentTarget
+        const formElements = form.elements as typeof form.elements & SelectInput
+        const categoryID: string = formElements.selectCategory.value
```

Comment:
I would save the selected category in a state when the user selects one then you can get it from there and dont need to extract it from the fromEvent

Reply:
I did that in the first step, but it made problems on the default value